tests/e2e/minio_deployment.yaml

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
This file is provided just for local testing, right?
As Minio is preinstalled by disconnected automation.

In such case, it may be located in dedicated folder, with instructions in https://github.com/project-codeflare/codeflare-sdk/blob/main/docs/e2e.md listing how to setup Minio.

Reply:
Also list there env variables which user can set.